packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -152,6 +152,11 @@ class TextFormField extends FormField<String> {
     SmartDashesType smartDashesType,
     SmartQuotesType smartQuotesType,
     bool enableSuggestions = true,
+    @Deprecated(
+        'Use autoValidateMode parameter which provide more specific '
+            'behaviour related to auto validation. '
+            'This feature was deprecated after v1.17.1'
```

Comment:
nit:indent and the tag number is inaccurate

Reply:
Sorry what do you mean by nit indent? this one is what the IDE is suggesting

Reply:
we do not completely follow the dart style guide, for the deprecation message we always indent by two spaces

Reply:
Ok